Bottom: bcc13794952108b393adfd5459c785032f53a178
Top:    a44cf247c4a28b24bd2788bf324fface588744c4
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-05 16:31:27 +0000

Remove left over debug


---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index c4077b0..a4ac5ba 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -1203,8 +1203,6 @@ object3d_from_copper_layers_within_area (POLYAREA *area)
 #endif
                               copper_appearance,
                               NULL));
-
-    //break;
   }
